phac-nml/irida-next
# Delete project

## What does this PR do and why?
Adding `Delete project` section under project `Advanced Settings` like described from #76.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/1e49c5a9-715f-4872-b502-7a215a221205)

## How to set up and validate locally
1. Navigate to a project settings page.
2. Hit the `Delete project` button under `Advanced Settings` in the `Delete project` section.
3. Verify the project is deleted.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
